test/sql/test_compare.py

```diff
@@ -1366,6 +1366,7 @@ def test_all_present(self):
                 or "inherit_cache" not in cls.__dict__
             )
             and not issubclass(cls, (Annotated))
+            and cls.__module__.split("/", 1)[0] != "test"
```

Comment:
how about a positive test here:  `cls.__module__.startswith("sqlalchemy.") `

Reply:
Good call!

Reply:
Works with testing for "sqlalchemy." on aarch64.